sdk/conftest.py

```diff
@@ -55,4 +70,125 @@ def clean_cached_resources():
         yield
         AbstractPreparer._perform_pending_deletes()
 except ImportError:
-    pass
\ No newline at end of file
+    pass
+
+
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
+def pytest_runtest_makereport(item, call) -> None:
```

Comment:
Do clients have to import this explicitly in their test files to ensure that this hook is registered as part of your test run?

Reply:
Nope! Since this is in the repo-wide `conftest.py` it'll automatically be applied unless it's overwritten elsewhere -- which is something I should double-check, to make sure that isn't done anywhere today.

I'm going to move the rest of the things I added to this `conftest.py` out into `devtools_testutils` though, since it came out to be a decent amount of code. I'll see if I can do an import in the file to still allow automatic availability to all tests

Reply:
Aha! I completely _completely_ missed the fact that this had been placed in `sdk/conftest.py`. Thanks McCoy!